initial-ai-src/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Ruter Reis Discount Calculator</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Ruter Reis Discount Calculator</h1>
+      <p>Find out what discount you can get on Oslo public transport tickets</p>
+    </header>
+
+    <main>
+      <div class="calculator-container">
+        <div class="input-section">
+          <label for="trips-per-week"
+            >How many trips do you expect per week?</label
+          >
+          <input
+            type="number"
+            id="trips-per-week"
+            min="0"
+            max="50"
+            value="10"
+          />
+
+          <label for="zone-selection">Which zones do you travel in?</label>
+          <select id="zone-selection">
+            <option value="1">Zone 1 (Oslo city center)</option>
+            <option value="2" selected>Zones 1-2 (Oslo + inner suburbs)</option>
+            <option value="3">Zones 1-3 (Oslo + outer suburbs)</option>
+            <option value="4">Zones 1-4 (Oslo + extended area)</option>
+            <option value="all">All zones (Oslo + Akershus)</option>
+          </select>
+
+          <label for="current-discount"
+            >What's your current Reis discount level?</label
+          >
+          <select id="current-discount">
+            <option value="0" selected>
+              0% - I'm starting fresh (first 4 tickets)
+            </option>
+            <option value="5">5% - I've bought 5-9 tickets recently</option>
```

Comment:
🎨 Maybe it's better to be explicit: "I've bought x tickets in the last 30 days". "Recently" is subjective.